engine/runtime/src/main/scala/org/enso/compiler/pass/resolve/VectorLiterals.scala

```diff
@@ -76,7 +76,10 @@ case object VectorLiterals extends IRPass {
     copyOfIr: T
   ): T = copyOfIr
 
-  private def vectorCons(bindings: BindingsMap): IR.Expression = {
+  private def vectorModule(bindings: BindingsMap): IR.Expression = {
+    val modName =
+      IR.Name.Literal(name = "Vector", isMethod = false, location = None)
```

Comment:
`VectorLiteralsTest` is failing because it cannot resolve `Vector` - maybe we could use FQN?